app/assets/javascripts/components/course/course.jsx

```diff
@@ -76,7 +76,7 @@ export const Course = withRouter(createReactClass({
   render() {
     const courseSlug = this.getCourseSlug();
     const course = this.props.course;
-    if (!courseSlug || !course || !course.home_wiki || course.title === '' || !this.props.users.length) { return <div />; }
```

Comment:
This doesn't seem right. Does this mean the whole Course component will just return an empty div if there are no users in the course?